config/locales/en.yml

```
@@ -954,6 +954,7 @@ en:
   price_groups:
     price_group_fields:
       internal_instruction: "Check this box if this price group is internal:"
+      hidden_instruction: "Check this box if you want to hide this price group:"
```

Maybe we should use another voice? Something like

```suggestion
      hidden_instruction: "Check this box if this price group is no longer used:"
```